src/main/java/com/google/sps/servlets/CreateGameServlet.java

```diff
@@ -43,35 +45,86 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/index.html");
     }
 
+    /**
+    * Retrieves the game's name.
+    * @param request the HttpServletRequest of the doPost.
+    * @return a String representing the name/title of the game.
+    */
     private String getName(HttpServletRequest request) {
         return request.getParameter("gameName");
     }
 
+    /**
+    * Retrieves the game's description.
+    * @param request the HttpServletRequest of the doPost.
+    * @return a String representing the description of the game.
+    */
     private String getDescription(HttpServletRequest request) {
         return request.getParameter("gameDescription");
     }
 
     // Below: TODO once game creation form is done
+    /**
+    * Retrieves the number of stages in the game.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an int representing the number of stages.
+    */
     private int getNumStages(HttpServletRequest request) {
         return -1;
     }
 
+    /**
+    * Retrieves the starting location of each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is a Coordinates representing
+    *         the starting location of the ith stage.
+    * @see com.google.sps.data.Coordinates
+    */
 	private ArrayList<Coordinates> getStageSpawnLocations(HttpServletRequest request) {
```

Comment:
Maybe put this function and the below functions in a separate class called "Stage" so that it matches the datastore. Then have a function that puts all of the stages in an array or maybe a next stage function that gets the next stage. This way you separate the stage and the game like how the datastore is layed out.

Reply:
I made the Stage class, but this function will still be necessary in later in order to set the properties of the Stage. I don't have enough information currently about how the datastore will work to generate a list of Stages yet.